tools/tools.yml

```diff
@@ -209,15 +209,78 @@
       site: https://github.com/andrea-cuttone/geoplotlib/wiki/User-Guide
       badges: pypi
 
-- name: Other domain-specific
-  intro: Tools focused on specific research or application areas other than those above.
+- name: Graphs and networks
+  intro: Tools to visualize graphs (networks).
   packages:
 
     - repo: networkx/networkx
       site: https://networkx.github.io
       appveyor_project: dschult/networkx-pqott
       badges: travis, appveyor, codecov, pypi, conda, site
 
+    - repo: skewed/graph-tool
+      # repo: https://git.skewed.de/count0/graph-tool
+      site: http://graph-tool.skewed.de
+      conda_channel: conda-forge
+      badges: conda, site
+
+    - repo: igraph/python-igraph
+      site: https://igraph.org/python
+      conda_channel: conda-forge
+      appveyor_project: ntamas/python-igraph
+      badges: travis, appveyor, pypi, conda, site
+
+    - repo: xflr6/graphviz
+      site: https://graphviz.readthedocs.io
+      conda_package: python-graphviz
+      conda_channel: conda-forge
+      badges: travis, codecov, pypi, conda, rtd, site
+
+    - repo: pygraphviz/pygraphviz
+      site: https://pygraphviz.github.io
+      conda_channel: conda-forge
+      badges: travis, pypi, conda, site
+
+    - repo: epfl-lts2/pygsp
+      site: https://pygsp.readthedocs.io
+      conda_channel: conda-forge
+      badges: travis, coveralls, pypi, conda, rtd, site
+
+    - repo: benmaier/netwulf
+      site: https://netwulf.readthedocs.io
+      badges: travis, pypi, rtd, site
+
+    - repo: cytoscape/py2cytoscape
+      site: https://py2cytoscape.readthedocs.io
+      conda_channel: conda-forge
+      badges: travis, pypi, conda, rtd, site
+
+    - repo: graphistry/pygraphistry
+      site: https://www.graphistry.com
+      badges: travis, pypi, rtd, site
```

Comment:
```suggestion
```

PyViz.org is a site for "open-source (OSS) Python data visualization tools", which I'd ordinarily interpret as "free and open-source" (though it's true we don't say so explicitly). From what I can see, graphistry is only usable with an API key and payment plan, so I don't think it qualifies. If anyone knows of a usable free version, please let us know, but otherwise I think it should be deleted from this list.